src/olympia/reviewers/utils.py

```diff
@@ -166,6 +167,7 @@ class Meta(PendingManualApprovalQueueTable.Meta):
 
     @classmethod
     def get_queryset(cls, request, **kw):
+        # Note: this must match what record_metrics() is doing.
```

Comment:
Add a reference to the file. `olympia.amo.cron.record_metrics` or something like that.